_posts/2019-09-05-nitty-gitty.md

```diff
@@ -0,0 +1,35 @@
+---
+title: The nitty-gitty
+summary: A web tech meetup for Brighton & Hove
+date: 2019-09-05T19:15:00
+image:
+  url: https://git-scm.com/images/logos/downloads/Git-Logo-2Color.png
+  title: Git logo
+  link: https://git-scm.com/downloads/logos
+speakers:
```

Comment:
should remi be in here?

Reply:
As discussed, ideally we'd have a "lightning speaker" section but for now I think it's best to leave as is